### PR TITLE
Brussels | ITP-2025-1 | Preeti Tyagi | Week 6 | Sprint 3 Coursework/sprint 3 implement and rewrite

### PR DESCRIPTION
**Pull request by preeti-t**
Self checklist

- [X] I have titled my PR with Region | Cohort | FirstName Last Name | Sprint | Assignment Title
- [X] My changes meet the requirements of the task
- [X] I have tested my changes
- [X] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

### CODE REVIEW

File: Sprint-3/1-implement-and-rewrite-tests/implement/2-is-proper-fraction.js [hunk 1]
```diff
@@ -46,14 +47,20 @@ assertEquals(improperFraction, false);
...
 const equalFraction = isProperFraction(3, 3);
-// ====> complete with your assertion
+assertEquals(equalFraction, false);
 
 // Stretch:
 // What other scenarios could you test for?
+// Input: numerator = 3, denominator = -8
+// target output: true
+// Explanation: The fraction 3/-8 is not a proper fraction because the numerator is equal to the denominator. The function should return true.
+const negativeDenominator = isProperFraction(3, -8);
```
Comment (MustafaBora) [inline Sprint-3/1-implement-and-rewrite-tests/implement/2-is-proper-fraction.js:64]: Good example, most doesn't look for absolute values. You also cover edge cases
